src/System.Management.Automation/engine/hostifaces/Connection.cs

```diff
@@ -700,6 +700,19 @@ internal System.Management.Automation.ExecutionContext ExecutionContext
             }
         }
 
+        /// <summary>
+        /// Gets the engine intrinsics.
+        /// </summary>
+        /// <exception cref="InvalidRunspaceStateException">Runspace is not opened.
+        /// </exception>
+        public System.Management.Automation.EngineIntrinsics EngineIntrinsics
+        {
+            get
```

Comment:
This can be shortened a little, perhaps?
```suggestion
            get => ExecutionContext?.EngineIntrinsics;
```
Also, would it be better to simply add a `using` clause for this file, if there isn't one for `System.Management.Automation`?

Reply:
Yes and yes. Both changes have been made.